aiven/client/cli.py

```diff
@@ -1317,9 +1317,9 @@ def service__integration_endpoint_create(self):
     @arg.json
     def service__integration_endpoint_update(self):
         """Update a service integration endpoint"""
+        endpoint_id = getattr(self.args, "endpoint-id")
```

Comment:
Fwiw I think it would be better to name the argument `endpoint_id` to avoid the use of `getattr()`.

Reply:
Makes sense, it's positional anyway. I'll change it...